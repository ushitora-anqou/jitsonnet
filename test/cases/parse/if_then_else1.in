if true then 1.0 else 2
