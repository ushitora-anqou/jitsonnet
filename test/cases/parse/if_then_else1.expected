{ Syntax.v =
  (Syntax.If
     ({ Syntax.v = Syntax.True;
        loc =
        { Syntax.fname = "test/cases/parse/if_then_else1.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 3 };
                  endpos = { Syntax.line = 1; column = 7 } })
          }
        },
      { Syntax.v = (Syntax.Number 1.);
        loc =
        { Syntax.fname = "test/cases/parse/if_then_else1.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 13 };
                  endpos = { Syntax.line = 1; column = 16 } })
          }
        },
      (Some { Syntax.v = (Syntax.Number 2.);
              loc =
              { Syntax.fname = "test/cases/parse/if_then_else1.in";
                ran =
                (Some { Syntax.startpos = { Syntax.line = 1; column = 22 };
                        endpos = { Syntax.line = 1; column = 23 } })
                }
              })));
  loc =
  { Syntax.fname = "test/cases/parse/if_then_else1.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 23 } })
    }
  }